src/mobile/package.json

```diff
@@ -115,7 +116,15 @@
     "react-addons-test-utils": "^15.6.2",
     "react-dom": "^16.2.0",
     "react-test-renderer": "16.6.1",
-    "rn-nodeify": "^8.2.0"
+    "rn-nodeify": "^10.0.1"
+  },
+  "resolutions": {
```

Comment:
Why are the `resolutions` necessary here? I think they should only be in the `shared` directory where these packages get installed.

Reply:
I think `yarn` started pulling in `1.0.0-beta8` dependencies for some reason. The resolutions seemed to fix that

Reply:
That's odd 🤔